frame/assets/Cargo.toml

```diff
@@ -36,7 +35,7 @@ version = "2.0.0"
 pallet-balances = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.16" }
 orml-tokens = { git = "https://github.com/open-web3-stack/open-runtime-module-library", rev = "2b1c9fb367ccb8e13601b2da43d1c5d9737b93c6", default-features = false }
 governance-registry = { package = "pallet-governance-registry", path = "../governance-registry", default-features = false }
-proptest = "0.9.6"
+proptest = "1.0"
```

Comment:
just curious if there is any specific requirement from version 1.0?

Reply:
it is about version to be same in all tests. other tests use 1.0